mathml/presentation-markup/tables/table-default-styles-001.html

```diff
@@ -21,6 +21,7 @@
   </math>
 </div>
 <script>
+document.fonts.load("25px Ahem").then(() => {
```

Comment:
Do we not need `explicit_done` on this test?

Reply:
As above - it hasn't seemed to be needed in my test runs, but I'll add it anyhow to be sure.